### PR TITLE
bumpNeighbours_ function moved to block_svg.

### LINKED ISSUE #1009: Return early from bumpNeighbours in headless mode
https://github.com/google/blockly/blob/develop/core/block.js#L333

### PR DESCRIPTION
**Pull request by marisaleung**
Fixed #1009

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly/1049)

### CODE REVIEW

File: core/block_svg.js [hunk 1]
```diff
@@ -1776,3 +1776,50 @@ Blockly.BlockSvg.prototype.getConnections_ = function(all) {
...
+          if (connection.isSuperior()) {
+            otherConnection.bumpAwayFrom_(connection);
+          } else {
+            connection.bumpAwayFrom_(otherConnection);
+          }
+        }
+      }
+    }
+  }
+};
```
Comment (rachel-fenichel) [inline core/block_svg.js]: Missing newline at end of file (that's what that symbol means).

File: core/block_svg.js [hunk 2]
```diff
@@ -1776,3 +1776,50 @@ Blockly.BlockSvg.prototype.getConnections_ = function(all) {
 Blockly.BlockSvg.prototype.makeConnection_ = function(type) {
   return new Blockly.RenderedConnection(this, type);
 };
+
+/**
+ * Bump unconnected blocks out of alignment.  Two blocks which aren't actually
+ * connected should not coincidentally line up on screen.
+ * @private
+ */
+// TODO: Refactor to return early in headless mode.
```
Comment (rachel-fenichel) [inline core/block_svg.js]: Remove todo.

**Comment by rachel-fenichel** [comment]
Two nits then lgtm.

File: core/block_svg.js (hunk 1 above)
Comment (marisaleung) [inline core/block_svg.js]: oooo cool. I thought the newlines needed to be deleted. Thank you!

File: core/block_svg.js (hunk 2 above)
Comment (marisaleung) [inline core/block_svg.js]: Done

**Pull request merged**